### PR TITLE
Added simple cleanup command

### PR DESCRIPTION
**Pull request by piaskowyk**
## Description

I do it often 😄 So I want to make it faster.

### CODE REVIEW

**Comment by tomekzaw** [comment]
Can we also remove Android build artefacts? More specifically, build and .cxx directories in android, Example/android and Example/android/app?

File: package.json [hunk 1]
```diff
@@ -25,7 +25,10 @@
     "type:generate:remove.ts": "find ./lib -type f -name \"*.ts\" -and -not -name \"*.d.ts\" -delete",
     "type:generate:remove.tsx": "find ./lib -type f -name \"*.tsx\" -and -not -name \"*.d.ts\" -delete",
     "prepare": "husky install",
-    "circular_dependency_check": "yarn madge --extensions js,ts,tsx --circular ./src"
+    "circular_dependency_check": "yarn madge --extensions js,ts,tsx --circular ./src",
+    "setup": "yarn && cd Example && yarn && cd ios && pod install && cd ../..",
```
Comment (jakub-gonet) [inline package.json:29]:
```suggestion
    "setup": "yarn && cd Example && yarn && cd ios && pod install --verbose && cd ../..",
```

If you have modules from docs installed `pod install` hangs, `--verbose` flag can hint what's going on.

**Review by jakub-gonet — COMMENTED** [review]
> Can we also remove Android build artifacts with yarn clean? More specifically, build and .cxx directories in android, Example/android and Example/android/app?

Preferably with another command, like `deep-clean` - building binary artifacts takes more time than installing node modules & pods.

File: package.json [hunk 2]
```diff
@@ -25,7 +25,12 @@
     "type:generate:remove.ts": "find ./lib -type f -name \"*.ts\" -and -not -name \"*.d.ts\" -delete",
     "type:generate:remove.tsx": "find ./lib -type f -name \"*.tsx\" -and -not -name \"*.d.ts\" -delete",
     "prepare": "husky install",
-    "circular_dependency_check": "yarn madge --extensions js,ts,tsx --circular ./src"
+    "circular_dependency_check": "yarn madge --extensions js,ts,tsx --circular ./src",
+    "setup": "yarn && cd Example && yarn && cd ios && pod install --verbose && cd ../..",
+    "clear": "rm -rf ./node_modules && cd Example && rm -rf ./node_modules && cd ios && pod deintegrate && cd ../..",
```
Comment (jakub-gonet) [inline package.json:30]:
```suggestion
    "clean": "rm -rf ./node_modules && cd Example && rm -rf ./node_modules && cd ios && pod deintegrate && cd ../..",
```

File: package.json [hunk 3]
```diff
@@ -25,7 +25,12 @@
...
+    "reset": "yarn clear && yarn setup",
+    "clean:deep": "cd android && rm -rf ./.cxx && rm -rf ./.gradle && rm -rf ./build && cd ../Example/android && rm -rf ./.gradle && rm -rf ./build &&rm -rf ./app/build && cd ../.. && yarn clear",
```
Comment (jakub-gonet) [inline package.json:32]:
```suggestion
    "clean:deep": "cd android && rm -rf ./.cxx && rm -rf ./.gradle && rm -rf ./build && cd ../Example/android && rm -rf ./.gradle && rm -rf ./build &&rm -rf ./app/build && cd ../.. && yarn clean",
```

File: package.json (hunk 2 above)
Comment (piaskowyk) [reply package.json]: Fixed 👍

File: package.json [hunk 4]
```diff
@@ -25,7 +25,12 @@
     "type:generate:remove.ts": "find ./lib -type f -name \"*.ts\" -and -not -name \"*.d.ts\" -delete",
     "type:generate:remove.tsx": "find ./lib -type f -name \"*.tsx\" -and -not -name \"*.d.ts\" -delete",
     "prepare": "husky install",
-    "circular_dependency_check": "yarn madge --extensions js,ts,tsx --circular ./src"
+    "circular_dependency_check": "yarn madge --extensions js,ts,tsx --circular ./src",
+    "setup": "yarn && cd Example && yarn && cd ios && pod install --verbose && cd ../..",
+    "clean": "rm -rf ./node_modules && cd Example && rm -rf ./node_modules && cd ios && pod deintegrate && cd ../..",
+    "reset": "yarn clean && yarn setup",
+    "clean:deep": "cd android && rm -rf ./.cxx && rm -rf ./.gradle && rm -rf ./build && cd ../Example/android && rm -rf ./.gradle && rm -rf ./build &&rm -rf ./app/build && cd ../.. && yarn clean",
```
Comment (tomekzaw) [inline package.json:32]:
```suggestion
    "clean:deep": "cd android && rm -rf ./.cxx && rm -rf ./.gradle && rm -rf ./build && cd ../Example/android && rm -rf ./.gradle && rm -rf ./build && rm -rf ./app/build && cd ../.. && yarn clean",
```

Comment (tomekzaw) [reply package.json:32]: Is `cd ../Example/android` correct after we've already used `cd android`? 🤔

**Review by tomekzaw — APPROVED** [review]
👏

**Pull request merged**